### PR TITLE
Added description and notebok for annotation consolidation demo

### PR DESCRIPTION
**Pull request by laurenmoos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ These examples provide quick walkthroughs to get you up and running with the lab
 - [Ground Truth Object Detection Tutorial](ground_truth_labeling_jobs/ground_truth_object_detection_tutorial) is a similar end-to-end example but for an object detection task.
 - [Basic Data Analysis of an Image Classification Output Manifest](ground_truth_labeling_jobs/data_analysis_of_ground_truth_image_classification_output) presents charts to visualize the number of annotations for each class, differentiating between human annotations and automatic labels (if your job used auto-labeling). It also displays sample images in each class, and creates a pdf which concisely displays the full results.
 - [Training a Machine Learning Model Using an Output Manifest](ground_truth_labeling_jobs/object_detection_augmented_manifest_training) introduces the concept of an "augmented manifest" and demonstrates that the output file of a labeling job can be immediately used as the input file to train a SageMaker machine learning model.
+- [Annotation Consolidation](annotation_consolidation) demonstrates Sagemaker Ground Truth annotation consolidation techniques  for image classification for a completed labeling job
```
Comment (natsirtguy) [inline README.md]:
```suggestion
- [Annotation Consolidation](annotation_consolidation) demonstrates Sagemaker Ground Truth annotation consolidation techniques for image classification for a completed labeling job.
```

File: ground_truth_labeling_jobs/README.md [hunk 2]
```diff
@@ -8,3 +8,4 @@ These examples provide quick walkthroughs to get you up and running with the lab
 - [Ground Truth Object Detection Tutorial](ground_truth_object_detection_tutorial) is a similar end-to-end example but for an object detection task.
 - [Basic Data Analysis of an Image Classification Output Manifest](data_analysis_of_ground_truth_image_classification_output) presents charts to visualize the number of annotations for each class, differentiating between human annotations and automatic labels (if your job used auto-labeling). It also displays sample images in each class, and creates a pdf which concisely displays the full results.
 - [Training a Machine Learning Model Using an Output Manifest](object_detection_augmented_manifest_training) introduces the concept of an "augmented manifest" and demonstrates that the output file of a labeling job can be immediately used as the input file to train a SageMaker machine learning model.
+- [Annotation Consolidation](annotation_consolidation) demonstrates Sagemaker Ground Truth annotation consolidation techniques  for image classification for a completed labeling job.
```
Comment (natsirtguy) [inline ground_truth_labeling_jobs/README.md]:
```suggestion
- [Annotation Consolidation](annotation_consolidation) demonstrates Amazon SageMaker Ground Truth annotation consolidation techniques for image classification for a completed labeling job.
```

File: README.md (hunk 1 above)
Comment (natsirtguy) [inline README.md]:
```suggestion
- [Annotation Consolidation](ground_truth_labeling_jobs/annotation_consolidation) demonstrates Amazon SageMaker Ground Truth annotation consolidation techniques for image classification for a completed labeling job.
```

File: ground_truth_labeling_jobs/README.md (hunk 2 above)
Comment (natsirtguy) [inline ground_truth_labeling_jobs/README.md]:
```suggestion
- [Annotation Consolidation](annotation_consolidation) demonstrates Amazon SageMaker Ground Truth annotation consolidation techniques for image classification for a completed labeling job.
```

**Comment by laurenmoos** [comment]
accepting changes

**Pull request merged**